include/clap/ext/draft/configurable-audio-ports.h

```diff
@@ -9,13 +9,9 @@ extern "C" {
 // This extension lets the host configure the plugin's input and output audio ports.
 // This is a "push" approach to audio ports configuration.
 static CLAP_CONSTEXPR const char CLAP_EXT_CONFIGURABLE_AUDIO_PORTS[] =
-   "clap.configurable-audio-ports.draft0";
+   "clap.configurable-audio-ports.draft1";
```

Comment:
@baconpaul do you feel like making a PR to define that convention in some file?

Reply:
I'm not sure what would be the right place to define it.
Maybe a `Conventions.md` at the root? :thinking:

Reply:
I propose that we do this file but in another PR.
There might be more things to put in that just the draft id syntax.

Let's not block this PR because of the ext id.